### ISSUE TITLE
Allow to use .explain() after .find()

### ISSUE THREAD

**Issue by TimPchelintsev** [ExtraBB/typegoose-cursor-pagination#36]
Thanks for this useful lib! As far as I understand, currently there is no way to use ```.explain()``` after internal .```find(...)``` call.
I want to debug my indexes usage, so it will be super helpful.

**Comment by ExtraBB** [comment]
I added a `findPagedExplain()` function for this purpose in `2.2.0`. Let me know if it does what you need.

**Comment by TimPchelintsev** [comment]
Yep, that will help for sure, thank you!

Honestly, I'll pasted code from this lib to my project directly and implemented this feature immediately after issue created.

I introduced another argument, ```explain```, to initial ```findPaged()``` function,
so that if it is presented, normal query is executed, than cloned and executed again with ```.explain()```, so that both query results  and explain results will be returned(under ```explain``` key).

I found such approach useful in automated api tests, when I can check both results correctness and index usage at the same time. Anyway, same functionality can be achieved by using brand new ```findPagedExplain()``` call alongside original one👍
